tools/release/rollback.sh

```diff
@@ -63,3 +64,9 @@ gcloud docker -- push gcr.io/${PROJECT_ID}/datalab-gateway:latest
 echo "Releasing the Datalab image: ${DATALAB_IMAGE}"
 docker tag -f ${DATALAB_IMAGE} gcr.io/${PROJECT_ID}/datalab:local
 gcloud docker -- push gcr.io/${PROJECT_ID}/datalab:local
+
+echo "Pulling the rollback GPU images: ${DATALAB_GPU_IMAGE}"
+gcloud docker -- pull ${DATALAB_GPU_IMAGE}
```

Comment:
The only graceful thing to do here is to exit.  As this is the last step in the rollback and the other images have already rolled back successfully, failing here will leave things in the desired state.